docs/how-tos/sensor-module.md

```diff
@@ -143,71 +160,28 @@ Run your test script from your terminal and make sure you are able to get readin
 There are a few standardized files that must be part of any module.
 You can create these automatically using the Viam module generator:
 
-{{< table >}}
-{{% tablestep %}}
-**1. Install the module generator**
-
-Install the [module generator](https://github.com/viam-labs/generator-viam-module/):
-
-1. Install node (version 16 or later) and npm if you don't already have them:
-
-   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
-   curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.3/install.sh | bash
-   nvm install 16
-   nvm use 16
-   ```
-
-1. Install Yeoman:
+1. Run the `module generate` command in your terminal:
 
    ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
-   npm install -g yo
+   viam module generate
    ```
 
-1. Now install the module generator:
-
-   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
-   npm install -g generator-viam-module
-   ```
-
-{{% /tablestep %}}
-{{% tablestep %}}
-**2. Run the generator**
-
-Go to the directory where you want to create your module and run the generator:
-
-```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
-yo viam-module
-```
-
-{{% /tablestep %}}
-{{% tablestep %}}
-**3. Name your model**
-
-When prompted for a model triplet, use `<your organization public namespace>:<repo name>:<what you want to call your sensor model>`.
-For example, `jessamy:weather:meteo_PM`.
-
-- You can find your organization namespace by going to your organization settings in the [Viam app](https://app.viam.com).
-- The repo name (also called family name) is generally the name of the GitHub repo where you will put your module code.
-  Name it something related to what your module does.
-- Name your sensor based on what it supports, for example, if it supports a model of ultrasonic sensor called "XYZ Sensor 1234" you could call your model `XYZ_1234` or similar.
-
-For more information, see [Name your new resource model](/how-tos/create-module/#name-your-new-resource-model).
-
-{{% /tablestep %}}
-{{% tablestep %}}
-**4. Specify the API**
-
-For the API triplet, enter `rdk:component:sensor`.
-This means that you are implementing the standard Viam sensor API.
-
-When asked whether this is a Viam SDK built-in API, enter `yes`.
+2. Follow the prompts, naming your module and selecting from the options.
 
-{{% /tablestep %}}
-{{< /table >}}
-<br>
+<!--prettier-ignore-->
+| Prompt | Description |
+| -------| ----------- |
+| Module name | The module name (also called repo name or family name) is generally the name of the GitHub repo where you will put your module code. Name it something related to what your module does. For example, `weather`. |
```

Comment:
It'd not also called repo name.
```suggestion
| Module name | The module name describes the module or the family of devices it supports. It is generally the same as the name of the GitHub repo where you will put your module code. For example, `weather`. |
```

Reply:
Okay. This is how it was previously referred to but I guess we can guide language here :)

Reply:
"Repo name" is indeed a confusing thing to call it and only some module repo names match module names--we should fix elsewhere too: https://viam.atlassian.net/browse/DOCS-3236